integration-cli/docker_api_inspect_test.go

```diff
@@ -42,3 +43,41 @@ func (s *DockerSuite) TestInspectApiContainerResponse(c *check.C) {
 		c.Fatalf("Path of `true` should not be converted to boolean `true` via JSON marshalling")
 	}
 }
+
+func (s *DockerSuite) TestInspectApiImageResponse(c *check.C) {
+	runCmd := exec.Command(dockerBinary, "pull", "busybox")
```

Comment:
How come we're pulling busybox here?
There is already a busybox loaded into the test env.